op-node/rollup/derive/span_channel_out.go

```diff
@@ -188,13 +220,36 @@ func (co *SpanChannelOut) AddSingularBatch(batch *SingularBatch, seqNum uint64)
 	return nil
 }
 
+func (co *SpanChannelOut) ensureOpenSpanBatch() {
+	if co.maxBlocksPerSpanBatch == 0 || co.spanBatch.GetBlockCount() < co.maxBlocksPerSpanBatch {
+		return
+	}
+	// we assume that the full span batch has been written to the last active rlp buffer
+	active, inactive := co.activeRLP(), co.inactiveRLP()
+	if inactive.Len() > active.Len() {
+		panic("inactive rlp unexpectedly larger")
+	}
```

Comment:
Was this something you encountered?

Reply:
Actually no, I just added this in an early implementation to assert correct code logic, but never hit this panic. We could remove it, given the new unit test coverage.

Reply:
fine to leave it in, was just curious